lib/moon_web/pages/components/tabs_page.ex

```diff
@@ -98,7 +98,16 @@ defmodule MoonWeb.Pages.Components.TabsPage do
     <Page {=@theme_name} {=@active_page} {=@breadcrumbs} {=@direction}>
       <ComponentPageDescription title="Tabs">
         <p>
-          Tabs
+          This component is already deprecated. Please use
```

Comment:
Style of the notification should be aligned with https://moon.io/components/tabs.